charts/incubator/blocky/values.yaml

```diff
@@ -0,0 +1,102 @@
+image:
+  repository: spx01/blocky
+  tag: development@sha256:c55e676e89cee31edeee687d70f7ed957b727d61b5611e213809f7a0399fe4ef
+  # repository: tccr.io/truecharts/blocky
+  # tag: v0.19@sha256:77a474542f12f480deca33ff0a6375846918b86988c13f858620839d8818ca84
+  pullPolicy: IfNotPresent
+
+imageWebUI:
+  repository: tccr.io/truecharts/blocky-frontend
+  tag: v0.0.3@sha256:81058f20520dcdb80c9883b6f21b338446fefc333e3ca8bd7d17336a24a5d842
+  pullPolicy: IfNotPresent
+
+env:
+  # Besides personal configuration, We also mount a configMap with some basic configs like redis, prometheus, paths for while/blacklists
+  # For this reason we instead use directory source for configs, if additional files (`configPath`, `whitelistPath` or `blacklistPath`) are defined
+
+  BLOCKY_CONFIG_FILE: '{{ ternary "/app/tc/" "/app/tc/tc-config.yaml" (ne .Values.blocky.configPath "")}}'
+
+blocky:
+  enableWebUI: true
+  enablePrometheus: true
+  configPath: ""
+  whitelistPath: ""
+  blacklistPath: ""
+
+probes:
+  liveness:
+    enabled:
+    custom: true
+    spec:
+      exec:
+        command:
+          - /app/blocky
+          - healthcheck
+  readiness:
+    custom: true
+    spec:
+      exec:
+        command:
+          - /app/blocky
+          - healthcheck
+  startup:
+    custom: true
+    spec:
+      exec:
+        command:
+          - /app/blocky
+          - healthcheck
+
+service:
+  main:
+    ports:
+      main:
+        port: 10315
+        protocol: HTTP
+        targetPort: 80
+  dns-tcp:
+    enabled: true
+    ports:
+      dns-tcp:
+        enabled: true
+        port: 53
+        targetPort: 53
+  dns-udp:
+    enabled: true
+    ports:
+      dns-udp:
+        enabled: true
+        port: 53
+        protocol: UDP
+        targetPort: 53
+  metrics:
+    enabled: true
+    ports:
+      metrics:
+        enabled: true
+        port: 4000
+        protocol: HTTP
+        targetPort: 4000
+
+redis:
+  enabled: true
+  existingSecret: "rediscreds"
+
+portal:
+  enabled: true
+# -- Blocky Whitelist content
+blockyWhitelist: ""
+# blockyWhitelist: |
+#   Whitelist here
+
+# -- Blocky Blacklist content
+blockyBlacklist: ""
+# blockyBlacklist: |
+#  # Blacklist here
```

Comment:
Found an issue:
With this design I cannot use internet blocklists/whitelists, which is the primary thing it's used with.

Simply put, there are 3 different sources for these:
- LocalFile
- ConfigMap
- URL